pyanaconda/dbus/constants.py

```diff
@@ -17,6 +17,8 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
+from pydbus.auto_names import auto_object_path
```

Comment:
It would be nice to have comment describing why we need it here. What's the reason.